backend/app/models.py

```diff
@@ -40,15 +42,62 @@ class TimeStampMixin(SQLModel):
     )
 
 
+class DistrictrMap(TimeStampMixin, SQLModel, table=True):
+    uuid: str = Field(sa_column=Column(UUIDType, unique=True, primary_key=True))
+    name: str | None = Field(nullable=False)
+    # This is intentionally not a foreign key on `GerryDBTable` because in some cases
+    # this may be the GerryDBTable but in others the pop table may be a materialized
+    # view of two GerryDBTables in the case of shatterable maps.
+    # We'll want to enforce the constraint tha the gerrydb_table_name is either in
+    # GerrydbTable.name or a materialized view of two GerryDBTables some other way.
+    gerrydb_table_name: str | None = Field(nullable=True)
+    # Null means default number of districts? Should we have a sensible default?
```

Comment:
(O) Sensible default hard to set globally, IMO, maybe later we are more geo-aware and can default to the num of congressional districts in a state but for now I think null is fine.